src/table/TableWrapper.jsx

```diff
@@ -66,7 +66,7 @@ export default function TableWrapper(props) {
   useEffect(() => {
     const updateSize = () => setTableWidth(rootElement.clientWidth);
     window.addEventListener('resize', updateSize);
-    tableSection.current.addEventListener('wheel', (evt) => handleScroll(evt, tableSection));
+    tableSection.current && tableSection.current.addEventListener('wheel', (evt) => handleScroll(evt, tableSection));
```

Comment:
What happens now in the return if we try to remove an event listener that does not exist?

Reply:
good question. It does not complain about anything. I tried to find a hint from documentation but find none.

Reply:
i tested that by directly replacing
`tableSection.current && tableSection.current.addEventListener('wheel', (evt) => handleScroll(evt, tableSection)); `
with
`tableSection.current.removeEventListener('wheel', (evt) => handleScroll(evt, tableSection));`

Reply:
ah ok we are fine then. I guess it doesn't do anything if it does fine the one it wants to delete